packages/sdk-js/src/plugins/third-party-tracking.ts

```diff
@@ -68,6 +68,25 @@ export function thirdPartyTrackingPlugin({
         promises.push(segmentPromise);
       }
 
+      if (trackers.includes("snowplow") && window.snowplow) {
```

Comment:
It's not super common but we do support names other than `snowplow`; not sure if that should be configurable or you just want to recommend using the generic `trackingCallback` in that case.

We also have the [Browser Tracker](https://docs.snowplow.io/docs/sources/trackers/javascript-trackers/web-tracker/tracker-setup/?platform=browser) which is an imported package so usually isn't exposed globally; that would likely also need a custom function to be able to call the equivalent `trackSelfDescribingEvent` method and may be worth documenting similar to your [Mixpanel example](https://docs.growthbook.io/lib/script-tag#mixpanel).